validphys2/src/validphys/pseudodata.py

```diff
@@ -369,15 +369,21 @@ def make_level1_data(data, level0_commondata_wc, filterseed, experiments_index,
 
     indexed_level1_data = indexed_make_replica(experiments_index, level1_data)
 
+    dataset_order = [cd.setname for cd in level0_commondata_wc]
+
     # ===== create commondata instances with central values given by pseudo_data =====#
     level1_commondata_dict = {c.setname: c for c in level0_commondata_wc}
     level1_commondata_instances_wc = []
 
+    # note that groupby sorts alphabetically
```

Comment:
We could add `sort=False` to groupby. AFAICT the behaviour is not documented, as keeping the order, so I would not touch the rest in any way.

Reply:
`sort=False` does not seem to solve the problem